cherryontech/jupiter-jumpers
# Add components

## Description

- Several updates to the styles for the stories, in order to make them responsive (making sizes 'rem'-based, and adding different root font sizes based on display size)
-  Update the header story to match our design
- Add error styling to the inputs (registration form); borders turn red and a red x appears when there is a validation error.

## Related Issue

N/A

## Related Story Card

N/A

## Type of Changes

Storybook updates

## Acceptance Criteria

N/A


## Testing Instructions

Check the updated stories here: https://662b31961b954f0f663b71e1-gscmnwueyk.chromatic.com/
- Registration form: when an invalid name or password is entered, the border should turn red, and a red x should appear. If you click the X, the field should clear for re-entry.
